controllers/Alerts.js

```diff
@@ -0,0 +1,93 @@
+const AlertRouter = require('express').Router();
+const mongodb = require('mongodb');
+const Alert = require('../models/Alert');
+
+// get a list of alerts
+AlertRouter.get('/list', function(req, res) {
+	Alert.find(function(err, CurrAlert) {
+		if (err) {
+			res.json(err);
+		}
+		const CurrAlert = CurrAlert.map(q => q.toJSONFor());
```

Comment:
Maybe change the naming here and use a new variable instead of CurrAlert? And I think we're using camel case for variables in general.

Reply:
Thank you! I was a bit confused with the cases because earlier Raph said that we will be using the CamelCaps instead of the camelBack but I'll make the changes right away! :D